src/mem/metaslab.h

```diff
@@ -91,6 +92,15 @@ namespace snmalloc
       head = (uint16_t)~0;
     }
 
+    void zero()
+    {
+      head = 0;
+      used = 0;
+      link = 0;
+      sizeclass = 0;
+      next = 0;
+    }
```

Comment:
It would be nice to have a `static_assert` on `sizeof(metaslab)` here so that if someone adds a field they don't silently break this.